src/index.js

```diff
@@ -0,0 +1,198 @@
+const cardsInfo = [
```

Comment:
Let's see if we can let JSON object in a different file.
I found this example
https://stackoverflow.com/questions/19706046/how-to-read-an-external-local-json-file-in-javascript